src/System.Drawing.Common/tests/Imaging/MetafileTests.cs

```diff
@@ -756,6 +757,7 @@ public void Ctor_NullPathI_ThrowsArgumentNullException()
             }
         }
 
+        [ActiveIssue(20844, TestPlatforms.AnyUnix)]
```

Comment:
Is this issue opened in CoreFX repo? Seems like #20844 is a PR.

Reply:
Should be #20884, I'll fix that.